projects/1.18/assets/gobber/gobber2/lang/zh_cn.json

```
@@ -0,0 +1,362 @@
+	"item.gobber2.gobber2_armor_end.line4": "靴子：在水中触发海豚的恩惠",
+	"item.gobber2.gobber2_armor_end.line5": "满装加成：可以在末地飞行和消除负面效果",
+	"item.gobber2.gobber2_helmet_end": "末影头盔",
+	"item.gobber2.gobber2_chestplate_end": "末影胸甲",
+	"item.gobber2.gobber2_leggings_end": "末影护腿",
+	"item.gobber2.gobber2_boots_end": "末影靴子",
+	"item.gobber2.gobber2_armor_dragon.line1": "拥有末影套装的所有能力",
+	"item.gobber2.gobber2_armor_dragon.line2": "满装加成：永久飞行和消除负面影响",
+	"item.gobber2.gobber2_armor_dragon.line3": "满装加成：负面效果抗性",
+	"item.gobber2.gobber2_helmet_dragon": "龙之头盔",
+	"item.gobber2.gobber2_chestplate_dragon": "龙之胸甲",
+	"item.gobber2.gobber2_leggings_dragon": "龙之护腿",
+	"item.gobber2.gobber2_boots_dragon": "龙之靴子",
+	"item.gobber2.gobber2_links_end": "末影装甲锁链",
+	"item.gobber2.gobber2_ring.cooldown": "冷却时间：%s",
+	"item.gobber2.gobber2_ring": "戈伯戒指",
+	"item.gobber2.gobber2_ring_nether": "下界戒指",
+	"item.gobber2.gobber2_ring_end": "末影戒指",
+	"item.gobber2.gobber2_ring_return": "回城戒指",
+	"item.gobber2.ring_return.line1": "欢迎回家！！",
+	"item.gobber2.ring_return.line2": "必须先右击床设置重生点",
+	"item.gobber2.ring_return.line3": "长按右击返回重生点",
+	"item.gobber2.ring_return.line4": "只能在主世界使用",
+	"item.gobber2.gobber2_ring_miner": "矿工戒指",
+	"item.gobber2.gobber2_ring_miner.line1": "破坏玩家周围的大部分原版方块",
+	"item.gobber2.gobber2_ring_miner.line2": "右击破坏方块，无掉落物",
+	"item.gobber2.gobber2_ring_miner.line3": "Shift右击破坏方块，有掉落物",
+	"item.gobber2.gobber2_ring_miner.line4": "右击破坏方块，有掉落物",
+	"item.gobber2.gobber2_ring_miner.line5": "Shift右击破坏方块，无掉落物",
+	"item.gobber2.gobber2_ring_miner.line6": "效果范围：11x5x11",
+	"item.gobber2.gobber2_ring_miner.mode.line1": "方块破坏模式：瞬间",
+	"item.gobber2.gobber2_ring_miner.mode.line2": "方块破坏模式：延迟",
+	"item.gobber2.gobber2_ring_lumberjack": "伐木工戒指",
+	"item.gobber2.gobber2_ring_lumberjack.line1": "破坏玩家周围的木头和树叶",
+	"item.gobber2.gobber2_ring_lumberjack.line2": "效果范围：11x40x11",
+	"item.gobber2.gobber2_ring_lumberjack.line3": "右击破坏方块，有掉落物",
+	"item.gobber2.gobber2_ring_lumberjack.line4": "Shift右击破坏方块，无掉落物",
+	"item.gobber2.gobber2_ring_farmer": "农民戒指",
+	"item.gobber2.gobber2_ring_farmer.line1": "适用于许多植物、农作物、树苗、甘蔗、瓜类和竹子",
+	"item.gobber2.gobber2_ring_farmer.line2": "Shift右击可种草和种花",
+	"item.gobber2.gobber2_ring_farmer.line3": "范围：12x12个方块",
+	"item.gobber2.gobber2_ring_acceleration": "加速戒指",
+	"item.gobber2.gobber2_ring_acceleration.line1": "当玩家在空中时提升速度",
+	"item.gobber2.gobber2_ring_acceleration.line2": "长按右击，然后跳起来",
+	"item.gobber2.gobber2_ring_acceleration.line3": "配置的速度值过高",
+	"item.gobber2.gobber2_ring_dismissal": "解雇戒指",
+	"item.gobber2.gobber2_ring_dismissal.line1": "可以把怪物扔出去",
+	"item.gobber2.gobber2_ring_dismissal.line2": "在玩家主手中时起作用",
+	"item.gobber2.gobber2_ring_attraction": "吸引戒指",
+	"item.gobber2.gobber2_ring_attraction.line1": "可以把掉落物吸过来",
+	"item.gobber2.gobber2_ring_attraction.line2": "吸引力： %s",
+	"item.gobber2.gobber2_ring_attraction.line3": "在物品栏中有效",
+	"item.gobber2.gobber2_ring_attraction.line4": "右击以切换开/关, 潜行+右击以知道吸引范围",
```

```suggestion
	"item.gobber2.gobber2_ring_attraction.line4": "右击以切换开/关, 潜行+右击以切换吸引范围",
```
原文`Right-click to toggle on/off, sneak + right-click to cycle through ranges` 和`知道`无关